scripts/base/frameworks/dpd/__load__.zeek

```diff
@@ -1 +1,3 @@
-@load ./main
+@deprecated "Remove in v6.1 - use @load base/frameworks/analyzer instead";
```

Comment:
Given that bare mode loads this anyways now, do we want to just say "Remove in v6.1 - now loaded automatically"?